roles/edpm_nftables/tasks/service-bootstrap.yml

```diff
@@ -42,6 +42,8 @@
         enabled: false
         state: stopped
         masked: true
+      when:
+        - ansible_facts.services["firewalld.service"] is defined
```

Comment:
let's also add ansible_facts.services[item]["status"] != "not-found", i recall weird cases without that